strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/gremlin/adapters/ArtifactAdapter.java

```diff
@@ -40,18 +40,25 @@
  * @author sbespalov
  */
 @Component
-public class ArtifactAdapter extends VertexEntityTraversalAdapter<Artifact> implements ArtifactHierarchyNodeAdapter<Artifact>
+public class ArtifactAdapter extends VertexEntityTraversalAdapter<Artifact>
+        implements ArtifactHierarchyNodeAdapter<Artifact>
 {
 
     @Inject
     GenericArtifactCoordinatesAdapter genericArtifactCoordinatesAdapter;
+
```

Comment:
@anki2189 could you not add empty lines here and always.

Reply:
@sbespalov I am using the formatter available on strongbox-documentation page and these lines are getting generated by it.

Reply:
@carlspring do we need this empty lines in formatter?

Reply:
Yeah, having an empty line between fields is how the majority of the code currently looks like.

Reply:
@carlspring will it be possible to review this rule? these empty lines don't bring any sense but from the other side its absence means a lot on my 13 inch laptop :)